app/bundles/InstallBundle/Tests/Install/InstallSchemaTest.php

```diff
@@ -22,14 +23,15 @@ class InstallSchemaTest extends \PHPUnit\Framework\TestCase
     public function setUp(): void
     {
         parent::setUp();
+        EnvLoader::load();
 
         $this->dbParams = [
-            'driver'        => getenv('DB_DRIVER') ?: 'pdo_mysql',
-            'host'          => getenv('DB_HOST'),
-            'port'          => getenv('DB_PORT'),
-            'dbname'        => getenv('DB_NAME'), // Doctrine needs 'dbname', not 'name'
-            'user'          => getenv('DB_USER'),
-            'password'      => getenv('DB_PASSWD'),
+            'driver'        => $_ENV['DB_DRIVER'] ?? 'pdo_mysql',
```

Comment:
@escopecz using `getenv` will always return false, Symfony stopped using `setenv` because it is not thread-safe. 

This is the only place where `$_ENV` is used as is